From c43daa5cdae91d1cfdab2e2e7994840fe358bdf7 Mon Sep 17 00:00:00 2001
From: kasi91 <kasi@experienceflow.com>
Date: Fri, 31 May 2019 07:30:25 +0000
Subject: [PATCH] New commit

---
 Helloworld.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Helloworld.py b/Helloworld.py
index 797a232..c87a1df 100644
--- a/Helloworld.py
+++ b/Helloworld.py
@@ -1,2 +1,2 @@
hi
helo
-print "H3llo World"
-print "Fr0m srcmake"
+print "Hello World"
+print "From srcmake"
-- 
2.7.4

